scipy/fft/_backend.py

```diff
@@ -0,0 +1,162 @@
+import scipy._lib.uarray as ua
+import scipy.fftpack as _fftpack
+from . import _pocketfft
+
+
+class _ScipyBackend:
+    """
+    The default backend for fft calculations
+    """
+    __ua_domain__ = "scipy.fft"
```

Comment:
This should be `numpy.scipy.fft`, as discussed in the hierarchical structure by @peterbell10, in case we update `uarray` to support that.

Reply:
```suggestion
    __ua_domain__ = "numpy.scipy.fft"
```

Reply:
Is their any issue with the fact that `numpy` and `scipy` share some functions with the same name but may be slightly incompatible?

Reply:
Then NumPy would use `"numpy.fft"`, SciPy would use `"numpy.scipy.fft"` and since none are submodules of the other, there's no issue. 😄

Reply:
But what about `__ua_domain__ = "numpy"`.

Reply:
We should change the documentation for this.

Reply:
When you decide what it should be, please add a comment explaining why it is what it is.

Reply:
> But what about __ua_domain__ = "numpy"

The multimethods are separate objects, so in theory there still shouldn't be an issue.